packages/editor/src/js/bindings/image-editor-cropper.js

```diff
@@ -217,29 +212,84 @@ export const EditorCropper = (editor) => {
      * Prevents user from resizing the selector outside and or larger than the image.
      * @param {any} oldBox - The old boundaries of the transformer as a rectangle.
      * @param {any} newBox - The new boundaries of the transformer as a rectangle.
+     * @param {any} ratio - The ratio selected for the crop by the user.
      * @returns - The rectangle that will be used as the new boundaries of the transformer.
      */
-    function handleSelectorResize(oldBox, newBox) {
+    function handleSelectorResize(oldBox, newBox, ratio) {
         const imageBox = image.getClientRect();
-        const imageW = image.width() * image.scaleX();
-        const imageH = image.height() * image.scaleY();
-
-        const minX = newBox.x + 1;
-        const minY = newBox.y + 1;
-        const maxX = minX + newBox.width - 2;
-        const maxY = minY + newBox.height - 2;
-
-        const isOut =
-            minX < imageBox.x ||
-            minY < imageBox.y ||
-            maxX > imageBox.x + imageW ||
-            maxY > imageBox.y + imageH ||
-            newBox.width > imageW ||
-            newBox.height > imageH;
-
-        if (isOut) return oldBox;
-
-        return newBox;
+    
+        // Calculates the image boundaries
+        const imageLeft = imageBox.x;
+        const imageTop = imageBox.y;
+        const imageRight = imageBox.x + imageBox.width;
+        const imageBottom = imageBox.y + imageBox.height;
+    
+        const overflowLeft = imageLeft - newBox.x;
+        const overflowTop = imageTop - newBox.y;
```

Comment:
// We can drag the selector outside the image. This calculates by how many pixels we went outside the image